docs/cow-protocol/tutorials/solvers/getting_new_routes_integrated_into_cow_protocol.md

```diff
@@ -0,0 +1,117 @@
+## Overview
+
+This guide is designed for teams building in DeFi who want to get their routes integrated and supported by CoW Protocol. Integration of new liquidity sources enables CoW Protocol's solvers to access more trading opportunities, potentially leading to better prices for users.
+
+## Why integrate with CoW Protocol?
+
+CoW Protocol's uses a unique batch auction mechanism and competition between multiple solvers.
+
+- Access to a growing user base of traders and integrators
```

Comment:
The first sentence of the section is not presented as a bullet point while all the rest do so, which probably looks a bit strange